src/tests/routers/generic/test_router_post.py

```diff
@@ -98,12 +105,27 @@ def test_no_platform_no_platform_resource_identifier(
     with logged_in_user():
         response = client_test_resource.post("/test_resources/v0", json=body, headers=headers)
     assert response.status_code == 200, response.json()
+
     body = {"title": "title2", "platform": None, "platform_resource_identifier": None}
     with logged_in_user():
         response = client_test_resource.post("/test_resources/v0", json=body, headers=headers)
     assert response.status_code == 200, response.json()
 
 
+def test_post_platform_and_platform_resource_identifier_rejected(
+    client_test_resource: TestClient
+):
+    headers = {"Authorization": "Fake token"}
+    body = {"title": "title1", "platform": "aiod", "platform_resource_identifier": 2}
+    with logged_in_user():
+        response = client_test_resource.post("/test_resources/v0", json=body, headers=headers)
+
+    assert response.status_code == 400
```

Comment:
```suggestion
    assert response.status_code == HTTPStatus.FORBIDDEN
```
(please check). I think `FORBIDDEN` is more apt than `BAD_REQUEST` since it is a valid request -- it's just that the user doesn't have the correct role.
Also -- I know this isn't done everywhere yet -- please use some kind of constant for status codes (such as `http.HTTPStatus`), then you don't have to remember which code is what :)

Reply:
That right. HTTPStatus.FORBIDDEN is the correct response.